### ISSUE TITLE
rework arkouda_metrics_exporter metrics retrieval

### ISSUE THREAD

**Issue by hokiegeek2** [Bears-R-Us/arkouda-contrib#68]
arkouda_metrics_exporter is currently reading pulling metrics via dedicated port on Arkouda in an effort to avoid blocking metrics request and therefore preserving a valid time-series data stream. Feedback from the Chapel team indicates this strategy is invalid b/c the dedicated port could still get blocked by a Chapel task. Consequently, need a strategy that does the following:

1. captures state of Arkouda server per completed Arkouda requests
2. is not blocked by Chapel processes

Options include the following:

1. read from file written to be Arkouda. This is the easiest, but depending upon whether the file location is on the same host (as opposed to a shared directory), it is possible Chapel processes could block.
2. read from Redis streams or Kafka. This is potentially a better approach as the Redis or Kafka source can be located anywhere and therefore is less likely to be blocked.

**Comment by hokiegeek2** [comment]
this is no longer needed since arkouda_metrics_exporter can be scraped at regular intervals.
